misc/config_tools/configurator/packages/configurator/src/pages/Config/ConfigForm/CustomWidget/CAT.vue

```diff
@@ -186,28 +217,27 @@ export default {
     },
   },
   watch: {
-    CAT_INFO: {
-      handler(newValue, _) {
-        if (newValue === null) {
-          // set formData CACHE_REGION to null
-          vueUtils.setPathVal(this.rootFormData, this.curNodePath, newValue)
-          return;
-        }
-        let data = [];
-        for (let i = 0; i < newValue.length; i++) {
-          data.push(newValue[i].data)
-        }
-        // set formData CACHE_REGION.CACHE_ALLOCATION to data
-        let CACHE_REGION = {CACHE_ALLOCATION: data}
-        vueUtils.setPathVal(this.rootFormData, this.curNodePath, CACHE_REGION)
-      },
-      deep: true
-    },
+    // CAT_INFO: {
+    //   handler(newValue, _) {
+    //     if (newValue === null) {
+    //       // set formData CACHE_REGION to null
+    //       vueUtils.setPathVal(this.rootFormData, this.curNodePath, newValue)
+    //       return;
+    //     }
+    //     let data = [];
+    //     for (let i = 0; i < newValue.length; i++) {
+    //       data.push(newValue[i].data)
+    //     }
+    //     // set formData CACHE_REGION.CACHE_ALLOCATION to data
+    //     let CACHE_REGION = {CACHE_ALLOCATION: data}
+    //     vueUtils.setPathVal(this.rootFormData, this.curNodePath, CACHE_REGION)
+    //   },
+    //   deep: true
+    // },
```

Comment:
Simply remove the code above if it is no longer needed.

Reply:
ok